### PR TITLE
Added TypeScript definitions

### PR DESCRIPTION
**Pull request by SmithAF**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "bellhop-iframe",
-  "version": "2.1.2",
+  "version": "2.1.3",
```
Comment (chipbell4) [inline package.json]: Let's mark this as a minor update, rather than patch (since it's a new feature and not a bugfix).

File: Bellhop.d.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export as namespace BellhopIframe;
+
+interface PriorityFunction extends Function {
+  private _priority: number;
```
Comment (andrewstart) [inline Bellhop.d.ts]: Interfaces usually don't allow member scoping - I'd be concerned that the compiler would complain about this when a project tries to build against it.

**Comment by andrewstart** [comment]
Oh, the `private` scopes in the classes were fine, it was just the one inside PriorityFunction that was a problem.

**Comment by SmithAF** [comment]
I kind of figured but I thought I would remove them anyways as they were mostly unnecessary anyways.

**Pull request merged**